### PR TITLE
[av][stories] add stories for expo-av

### PR DESCRIPTION
**Pull request by ajsmth**
# Why

Another example of more targeted stories / components for expo-av

https://user-images.githubusercontent.com/40680668/132394419-cbfc47e1-191d-417f-a3f3-ac3bf27dca60.mov

 Test Plan

- navigate to `packages/expo-av` and run `yarn examples:start`

### CODE REVIEW

File: packages/expo-av/package.json [hunk 1]
```diff
@@ -1,41 +1,47 @@
...
+	},
+	"devDependencies": {
+		"expo-module-scripts": "^2.0.0"
+	},
+	"expoStories": {
+		"packages": {
+			"expo-av": "~9.2.0"
+		}
+	}
+}
```
Comment (bbarthec) [inline packages/expo-av/package.json:47]: Please revert indent changes in this file. We aim to indent everything with 2 spaces in every file kind.

File: packages/expo-av/src/__stories__/Audio.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+          renderControls={props => <VolumeControls {...props} />}
+        />
+      </Container>
+
+      <Container labelTop="Set Looping">
+        <AudioPlayer
+          source={{ uri: remoteUrl }}
+          renderControls={props => <LoopingControls {...props} />}
+        />
+      </Container>
```
Comment (bbarthec) [inline packages/expo-av/src/__stories__/Audio.stories.tsx:73]:
```suggestion
      {
        [
          {
            label: "Play Pause and Stop Audio",
            ControlsComponent: PlayPauseStopControls,
          },
          {
            label: "Skip Forwards and Backwards",
            ControlsComponent: SkipControls,
          },
          {
            label: "Set Playback Rates",
            ControlsComponent: PlaybackRateControls,
          },
          {
            label: "Set Volume",
            ControlsComponent: VolumeControls,
          },
          {
            label: "Set Looping",
            ControlsComponent: LoopingControls,
          },
        ].map(({ label, ControlsComponent }) => (
          <Container labelTop={label} key={label}>
            <AudioPlayer
              source={{ uri: remoteUrl }}
              renderControls={props => <ControlsComponent {...props} />}
            />
          </Container>
        ))
      }
```

File: packages/expo-av/tsconfig.json [hunk 3]
```diff
@@ -1,9 +1,15 @@
...
+	},
+	"include": [
+		"./src"
+	],
+	"exclude": [
+		"**/__mocks__/*",
+		"**/__tests__/*",
+		"**/__stories__/*"
+	]
+}
```
Comment (bbarthec) [inline packages/expo-av/tsconfig.json:15]: This file should be indented with 2 spaces.

**Review by bbarthec — COMMENTED** [review]
I think it's good, but try to limit boilerplate code when there's 3 or more parts looking almost identical 🤔 
I've suggested one of such refactor, but it applies to any other Component you've written.

**Review by lukmccall — APPROVED** [review]
LGTM ✅

**Review by bbarthec — APPROVED** [review]

**Pull request merged**